src/content/docs/en/guides/content.mdx

```diff
@@ -4,7 +4,7 @@ description: >-
   Astro is a perfect choice for your content-focused site: blogs, marketing
   sites, portfolios, and more! Author your content directly in your project, or
   connect your CMS of choice.
-i18nReady: false
+i18nReady: true
```

Comment:
Also, this one is not really discoverable yet, and I might want to use it for other purposes. Great call liruifengv!
```suggestion
i18nReady: false
```